packages/devices/origin-device-registry-irec-local-api/src/device/device.module.ts

```diff
@@ -1,27 +1,37 @@
 import { Module } from '@nestjs/common';
 import { CqrsModule } from '@nestjs/cqrs';
 import { TypeOrmModule } from '@nestjs/typeorm';
-import { ConnectionModule, RegistrationModule } from '@energyweb/origin-organization-irec-api';
 import { ConfigModule } from '@nestjs/config';
 import { PassportModule } from '@nestjs/passport';
+import { ScheduleModule } from '@nestjs/schedule';
+
+import {
+    ConnectionModule,
+    IrecModule,
+    RegistrationModule
+} from '@energyweb/origin-organization-irec-api';
+import { UserModule } from '@energyweb/origin-backend';
 
 import { DeviceController } from './device.controller';
 import { Device } from './device.entity';
 import { DeviceService } from './device.service';
 import { ValidateDeviceOwnershipCommandHandler } from './handlers/validate-device-ownership.handler';
-import { IrecDeviceService } from './irec-device.service';
+import { CheckDeviceStateTask } from './cron';
 
 @Module({
     imports: [
+        ScheduleModule.forRoot(),
```

Comment:
I guess this should be initialized in root module. As it is "forRoot" 😆 

No, but TBH I'm not sure if now the scheduling system will be available in neighbour module branches. I guess this should really by in root module

Reply:
root module of the package*

Reply:
done